frontend/public/components/namespace.jsx

```diff
@@ -264,20 +265,19 @@ const getNamespacesSelectedColumns = () => {
 
 const namespacesRowStateToProps = ({ UI }) => ({
   metrics: UI.getIn(['metrics', 'namespace']),
-  selectedColumns: UI.getIn(['columnManagement']),
 });
 
 const NamespacesTableRow = connect(namespacesRowStateToProps)(
-  withTranslation()(({ obj: ns, index, key, style, metrics, selectedColumns, t }) => {
+  withTranslation()(({ obj: ns, index, key, style, metrics, t }) => {
     const name = getName(ns);
     const requester = getRequester(ns);
     const bytes = metrics?.memory?.[name];
     const cores = metrics?.cpu?.[name];
     const description = getDescription(ns);
     const labels = ns.metadata.labels;
-    const columns = new Set(
-      selectedColumns?.get(NamespacesColumnManagementID) || getNamespacesSelectedColumns(),
-    );
+    const [tableColumns, , loaded] = useTableColumns(NamespacesColumnManagementID);
+    const columns =
+      loaded && tableColumns?.length > 0 ? new Set(tableColumns) : getNamespacesSelectedColumns();
```

Comment:
You cannot render the table until the settings have loaded otherwise there's potential for the table to flash the default columns.
Same comment for all usages in this PR.

Reply:
Done